keps/prod-readiness/sig-cli/2227.yaml

```diff
@@ -0,0 +1,3 @@
+kep-number: 2227
+alpha:
+  approver: "@johnbelamaric"
```

Comment:
if you get to it before your PR is lgtm'd by sig-cli, this ought to be @deads2k

Reply:
i'll wait with tagging this for that having fixed.